snowflake_notebooks/1_sf_nb_snowpark_ml_data_ingest.ipynb

```diff
@@ -0,0 +1,476 @@
+{
+ "metadata": {
+  "kernelspec": {
+   "display_name": "pysnowpark",
+   "language": "python",
+   "name": "python3"
+  },
+  "language_info": {
+   "codemirror_mode": {
+    "name": "ipython",
+    "version": 3
+   },
+   "file_extension": ".py",
+   "mimetype": "text/x-python",
+   "name": "python",
+   "nbconvert_exporter": "python",
+   "pygments_lexer": "ipython3",
+   "version": "3.11.7"
+  },
+  "orig_nbformat": 4
+ },
+ "nbformat_minor": 2,
+ "nbformat": 4,
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "437b8d75-4401-4ae3-b1d9-1fb7496c225c",
+   "metadata": {
+    "name": "CELL20",
+    "collapsed": false
+   },
+   "source": "# Select Packages\n\nTo get started, let's select a few packages that we will need. In the **Packages** drop-down picker in the top right of the UI, search for and add the following packages by clicking on them:\n\n- snowflake-ml-python\n\nOnce you add the packages, click the **Start** button! Once it says **Active**, you're ready to run the rest of the Notebook."
```

Comment:
Do we expect this to also work on SPCS? There is no package picker and we will already have snowflake-ml installed in the image. But that's not PuPr yet so I guess this is just for WH